src/components/unique-token/UniqueTokenImage.js

```diff
@@ -102,12 +103,13 @@ const UniqueTokenImage = ({
               onError={handleError}
               onLoad={onLoad}
               resizeMode={ImgixImage.resizeMode[resizeMode]}
+              size={size}
               source={{ uri: image }}
               style={position.coverAsObject}
             />
             {!loadedImg && lowResUrl && (
```

Comment:
now this always false no?

Reply:
im not following, this was the underlying issue. without passing size to imgix we never ask it to resize the image. ive added to almost every case but if its undefined we dont pass to imgix already so this should be fine